ofono2mm/mm_modem.py

```diff
@@ -120,13 +128,122 @@ async def init_mm_3gpp_interface(self):
         self.bus.export('/org/freedesktop/ModemManager1/Modem/' + str(self.index), self.mm_modem3gpp_interface)
         self.mm_modem3gpp_interface.set_props()
 
+    async def init_mm_simple_interface(self):
+        self.mm_modem_simple_interface = MMModemSimpleInterface(self)
+        self.bus.export('/org/freedesktop/ModemManager1/Modem/' + str(self.index), self.mm_modem_simple_interface)
+
     async def init_mm_messaging_interface(self):
         self.mm_modem_messaging_interface = MMModemMessagingInterface(self.index, self.bus, self.ofono_proxy, self.modem_name, self.ofono_modem, self.ofono_props, self.ofono_interfaces, self.ofono_interface_props)
         self.bus.export('/org/freedesktop/ModemManager1/Modem/' + str(self.index), self.mm_modem_messaging_interface)
         if 'org.ofono.MessageManager' in self.ofono_interfaces:
             self.mm_modem_messaging_interface.set_props()
             await self.mm_modem_messaging_interface.init_messages()
 
+    async def check_ofono_contexts(self):
+        global bearer_i
+        if not 'org.ofono.ConnectionManager' in self.ofono_interfaces:
+            return
+        contexts = await self.ofono_interfaces['org.ofono.ConnectionManager'].call_get_contexts();
+        old_bearer_list = self.props['Bearers'].value
+        for ctx in contexts:
+            if ctx[1]['Type'].value == "internet":
+                mm_bearer_interface = MMBearerInterface(self.index, self.bus, self.ofono_proxy, self.modem_name, self.ofono_modem, self.ofono_props, self.ofono_interfaces, self.ofono_interface_props, self)
```

Comment:
Super-nitpick: could this be split in newlines so that it's easier to read? Like
```
mm_bearer_interface = MMBearerInterface(
    self.index,
    self.bus,
    self.ofono_proxy,
    ...
    self
)
```